src/screens/CreatePost/index.js

```diff
@@ -322,7 +323,6 @@ const CreatePost = () => {
       asset.append('photo', composeImageMeta(compressionResult));
 
       const responseUpload = await uploadPhoto(asset);
-
       setMediaStorage((val) => [...val, newArr]);
       setDataImage((val) => [...val, responseUpload.data.url]);
       sheetMediaRef.current.close();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The response from the `uploadPhoto` function is now being stored in `responseUpload`. This is a good practice as it allows for potential error handling or further use of the response data.
```diff
- const responseUpload = await uploadPhoto(asset);
+ const responseUpload = await uploadPhoto(asset);
+ setMediaStorage((val) => [...val, newArr]);
+ setDataImage((val) => [...val, responseUpload.data.url]);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->